src/app/lib/config/services/arabic.js

```diff
@@ -203,7 +203,8 @@ export const service = {
     footer: {
       trustProjectLink: {
         href: 'https://www.bbc.com/arabic/institutional-49283069',
-        text: 'لماذا يمكنك الإعتماد على أخبار بي بي سي',
+        text:
+          'لماذا يمكنك الإعتماد على أخبار بي بي سيلماذا يمكنك الاعتماد على أخبار بي بي سي',
```

Comment:
Is this one correct? It seems to be duplicated.

Reply:
Good point. This may have been a mistake in the spreadsheet. I will revert this change for now and add Arabic updates to another PR later on.